README.md

```diff
@@ -1,8 +1,13 @@
 # MLab Visualizations
 
-[![Build Status](https://travis-ci.org/bocoup/mlab-vis-client.svg?branch=master)](https://travis-ci.org/bocoup/mlab-vis-client)
+[![Build Status](https://travis-ci.org/opentechinstitute/mlab-vis-client.svg?branch=mlab-staging)](https://travis-ci.org/opentechinstitute/mlab-vis-client)
 
```

Comment:
Ok. I'll update this call to Travis so it's reading from the correct branch. This got pulled in when I rebased from your staging branch.